doc/source/example-newsreader.rst

```diff
@@ -0,0 +1,27 @@
+News Reader
+===========
+
+This document shows how to implement a simple news reader using Ray. The reader
+consists of a simple Vue.js `frontend`_ and a backend consisting of a Flask
+server and a Ray actor. View the `code for this example`_.
+
+To run this example, you will need to install NPM and a few python dependencies.
+
+.. code-block:: bash
+
+  pip install atoma
+  pip install flask
+
+
+To use this example you need to
+
+* Start the server with ``python server.py``
```

Comment:
Make it clear which directory the user needs to be in to run this.

Reply:
done